.github/workflows/pythonpackage.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         sudo sysctl -w vm.swappiness=1
         sudo sysctl -w fs.file-max=262144
         sudo sysctl -w vm.max_map_count=262144
-        ES_VERSION=6.6.2 ES_DOWNLOAD_URL=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz
+        ES_VERSION=7.13.2-linux-x86_64 ES_DOWNLOAD_URL=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz
```

Comment:
when unpacking the folder name does not contain the architecture.

I would leave the version to `7.13.2` and add the architecture to the url `${ES_VERISON}-linux-x86_64`